modules/dumbCommandModule/index.js

```diff
@@ -57,7 +57,9 @@ module.exports.commands = {
     },
     add: function(r, parts, reply, command, from) {
       if(parts.length !== 2) return reply("add must have *exactly* two arguments");
-      if(!allowCmds && parts[1].charAt(0) === '!') return reply ("You are an asshole");
+      if(!allowCmds && parts[1].charAt(0) === '!') return reply("You are an asshole");
```

Comment:
I'd personally find `if(!allowCmds && (/^!/m).test(parts[1]))` slightly more readable than expanding it to three checks I think, but regex readability is totally a matter of taste thing.

Reading this now, I realize that there's technically `bot.config.commandPrefix` meaning this isn't strictly a correct check since the prefix isn't necessarily `!`.. but this is no worse than it was before.

I'll leave it up to you whether you want to tackle that or not. If you want to, [here's](https://github.com/euank/EuIrcBot/blob/3d91a5cbf0255320656f9e7cb39c5da5cc2ac433/bot.js#L216-L226) the command parsing logic, which is a good reference.

It's possible it should be in a suitably exposed function such that this module can share it with `bot.js`.